src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,80 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static final List dukeList = new List();
+    public static final String markDone = "mark";
+    public static final String bye = "bye";
+    public static final String list = "list";
+    public static final String space = " ";
+    public static final String unmarkDone = "unmark";
+
+    public static void main(String[] args) {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        Message.printGreeting();
+        echo();
+    }
+
+    public static void echo() {
+        String line;
+        Scanner in = new Scanner(System.in);
+        line = in.nextLine();
+        while (!line.equals(bye)) {
+            try {
+                translateInput(line);
+            } catch (DukeException e) {
+                Message.printError();
+            }
+            line = in.nextLine();
+        }
+        Message.printingExit();
+    }
+
+    public static int checkInteger(String[] wordsInput) throws DukeException {
+        if (wordsInput.length != 2 || wordsInput[1].equals("")) {
+            throw new DukeException();
+        }
+        int taskNumber;
+        try {
+            taskNumber = Integer.parseInt(wordsInput[1]);
+        } catch (NumberFormatException e) {
+            throw new DukeException();
+        }
+        if (taskNumber > dukeList.getListSize()) {
```

Comment:
make you can also put this in the try, since you are throwing an exception as well (might make it easier to read). not necessary at all though